app/customer.py

```diff
@@ -0,0 +1,47 @@
+from math import sqrt
```

Comment:
The import statement for 'sqrt' is correct, but ensure you use exactly 'import datetime' as per the checklist. This line is not related to datetime, so no changes are needed here.